qemu/qemus390x.go

```diff
@@ -63,6 +63,31 @@ var isVirtioPCI = map[DeviceDriver]bool{
 	PCIePCIBridgeDriver: false,
 }
 
+var isVirtioCCWMap = map[DeviceDriver]bool{
+	NVDIMM:              false,
+	Virtio9P:            true,
+	VirtioNetCCW:        true,
+	VirtioSerial:        true,
+	VirtioBlock:         true,
+	VirtioBlockCCW:      true,
+	Console:             false,
+	VirtioSerialPort:    false,
+	VHostVSock:          true,
+	VirtioRng:           true,
+	VirtioBalloon:       true,
+	VhostUserSCSI:       false,
+	VhostUserBlk:        false,
+	Vfio:                true,
+	VirtioScsi:          true,
+	PCIBridgeDriver:     false,
+	PCIePCIBridgeDriver: false,
+}
+
+// isVirtioCCW returns if the device is a ccw device
+func isVirtioCCW(device DeviceDriver) bool {
```

Comment:
What's the purpose of this function?  There's nothing wrong with it but it is inconistent with the way the checks are handled for the pci driver, where we check the map directly, e.g.,

```
	if isVirtioPCI[fsdev.Driver] {
		deviceParams = append(deviceParams, fmt.Sprintf(",romfile=%s", fsdev.ROMFile))			deviceParams = append(deviceParams, fmt.Sprintf(",romfile=%s", fsdev.ROMFile))
	}		}
	if isVirtioCCW(fsdev.Driver) {
```

whichlooks a little weird.

Reply:
The purpose of the function is to return false on not-s390x arch. It was a little bit smaller than having a mapping as for isVirtioPCI, but if you prefer for consistency, I can create a mapping in qemu_arch_base.go with false for all the devices.

Reply:
The function identifies the devices that needs the devno parm

Reply:
My Go's a little rusty these days but couldn't you achieve the same by doing

var isVirtioCCW = map[DeviceDriver]bool{}

in qemu_arch_base.go?

Reply:
which would be even smaller

Reply:
ok, no problem. I'm going to update the code accordingly

Reply:
I changed the function to map